src/services/PluginService.ts

```diff
@@ -1,5 +1,8 @@
-import { apiClient, tryRequest } from '@/api'
+
 import type { JSONSettings } from '@models/JSONSchema'
+import { useApiClient } from '../composables/useApiClient'
```

Comment:
Vite has the auto-imports plugin, no need for this explicit import.

Reply:
import from `@composables/...` was not working, I'll check maybe I need to ocnfigure something